package.json

```diff
@@ -80,8 +80,8 @@
     "dev-build": "webpack --progress --mode development",
     "dev-build-test": "npm run dev-build && npm run test",
     "dev-start": "webpack-dev-server --config ./webpack.config.js --mode development",
-    "eslint": "eslint --max-warnings 7 --color -c .eslintrc.js --ext js,jsx .",
-    "eslint-fix": "eslint --max-warnings 7 --color -c .eslintrc.js --ext js,jsx --fix .",
+    "eslint": "eslint --max-warnings 8 --color -c .eslintrc.js --ext js,jsx .",
+    "eslint-fix": "eslint --max-warnings 8 --color -c .eslintrc.js --ext js,jsx --fix .",
```

Comment:
Note: I had to make this change as the "complexity" configuration was apparently being ignored, and producing a warning on the `reducers/index.js` switch statement.